.env.dist.testing

```diff
@@ -8,7 +8,7 @@ TEST_SITE_ADMIN_USERNAME=admin
 TEST_SITE_ADMIN_PASSWORD=password
 TEST_SITE_ADMIN_EMAIL="wordpress@convertkit.com"
 TEST_SITE_WP_ADMIN_PATH=/wp-admin
-WP_ROOT_FOLDER="/src"
+WP_ROOT_FOLDER="/home/runner/work/convertkit-woocommerce/convertkit-woocommerce/wordpress"
```

Comment:
This (and the rest of the env changes) should probably not be checked in?

Reply:
`.env.dist.testing` must be checked in, as it's used by Codeception when running acceptance and unit tests, both of which are run automatically by our [GitHub action](https://github.com/ConvertKit/convertkit-woocommerce/actions/runs/1952925522) to confirm that a PR passes tests before being merged into the main branch.

`.env.example` is provided as a convenience when setting up your local development and testing environment: https://github.com/ConvertKit/convertkit-wordpress/blob/master/SETUP.md#configure-testing-environment

Neither contain sensitive information; API keys for testing are injected by using GitHub's repository secrets (https://github.com/ConvertKit/convertkit-woocommerce/settings/secrets/actions).

Hope that helps.

Reply:
Oh, what I meant was that I thought e.g. the `WP_ROOT_FOLDER` should remain `/src` and not a machine-specific folder

Reply:
Ah, no - `.env.dist.testing` will always specify the machine-specific folder, as that's where the GitHub action places the WordPress installation files. Our tests read this environment file to know where the installation is.

For local testing, you'd use your own `.env.testing` file when setting up your local development and testing environment: https://github.com/ConvertKit/convertkit-wordpress/blob/master/SETUP.md#configure-testing-environment.

These environment files (and others) are excluded when we deploy Plugin updates to https://wordpress.org/plugins/convertkit-for-woocommerce/, as defined by the `.distignore` file: https://github.com/ConvertKit/convertkit-woocommerce/blob/master/.distignore

Reply:
thank you!